apps/gauzy/src/app/@theme/components/theme-sidebar/theme-settings/theme-settings.component.ts

```diff
@@ -1,22 +1,40 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy, AfterViewChecked } from '@angular/core';
 import { NbSidebarService } from '@nebular/theme';
+import { tap } from 'rxjs/operators';
 
```

Comment:
@Gramosx  add this decorator in order to unsubscribe inside the pipe
```suggestion
@UntilDestroy({ checkProperties: true }
```